web/docs/project/env-vars.md

```diff
@@ -2,61 +2,165 @@
+
 **Environment variables** are used to configure projects based on the context in which they run. This allows them to exhibit different behaviors in different environments, such as development, staging, or production.
 
-For instance, _during development_, you may want your project to connect to a local development database running on your machine, but _in production_, you may prefer it to connect to the production database. Similarly, in development, you may want to use a test Stripe account, while in production, your app should use a real Stripe account.
+For instance, _during development_, you may want your project to connect to a local development database running on your machine, but _in production_, you want it to connect to the production database. Similarly, in development, you may want to use a test Stripe account, while in production, your app should use a real Stripe account.
 
 While some env vars are required by Wasp, such as the database connection or secrets for social auth, you can also define your env vars for any other useful purposes, and then access them in the code.
 
-In Wasp, you can use environment variables in both the client and the server code.
+Let's go over the available env vars in Wasp, how to define them, and how to use them in your project.
+
 ## Client Env Vars
 
-Client environment variables are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys -> you can provide those to server instead).
+Client environment variables are injected into the client Javascript code during the build process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys, you should store secrets in the server env variables).
 
-To enable Wasp to pick them up, client env vars must be prefixed with `REACT_APP_`, for example: `REACT_APP_SOME_VAR_NAME=...`.
+<ClientEnvVarsNote />
 
 You can read them from the client code like this:
 
 <Tabs groupId="js-ts">
 <TabItem value="js" label="JavaScript">
 
 ```js title="src/App.js"
-console.log(import.meta.env.REACT_APP_SOME_VAR_NAME)
+import { env } from 'wasp/client'
+
+console.log(env.REACT_APP_SOME_VAR_NAME)
 ```
 </TabItem>
 <TabItem value="ts" label="TypeScript">
 
 ```ts title="src/App.ts"
-console.log(import.meta.env.REACT_APP_SOME_VAR_NAME)
+import { env } from 'wasp/client'
+
+console.log(env.REACT_APP_SOME_VAR_NAME)
 ```
 </TabItem>
 </Tabs>
 
+Read more about the `env` object in the [API reference](#client-env-vars-1).
 
-Check below on how to define them.
+### Wasp Client Env Vars
 
+Here are the client env vars that Wasp defines:
+
+#### General Configuration
+
+<EnvVarsTable>
+  <EnvVar name="REACT_APP_API_URL" type="URL" isRequired={false} defaultValue="http://localhost:3001" note="The client uses this as the server URL." />
+</EnvVarsTable>
 
```

Comment:
Should we mention that this is it, only one? Somehow it feels like waht now, where is the rest. Maybe we can mention that this is it but you can also deifne your own. Ok maybe that is silly, I am not sure, but this somehow felt unfinished.

Reply:
It's following the same structure as the server env vars, I think this is understandable.